tests/test_ote_api.py

```diff
@@ -320,12 +330,64 @@ def progress_callback(progress: float, score: Optional[float] = None):
         self.assertTrue(np.all(training_progress_curve[1:] >= training_progress_curve[:-1]))
 
     @e2e_pytest_api
-    def test_inference_progress_tracking(self):
+    def test_nncf_optimize_progress_tracking(self):
+        if not is_nncf_enabled:
```

Comment:
`is_nncf_enabled` is a function. Please make sure, that tests work w/o NNCF installed.

Reply:
done